### PR TITLE
update advanced api flow example

### PR DESCRIPTION
**Pull request by immavalls**
Fixes https://github.com/grafana/k6-docs/issues/1170

Preview: https://mdr-ci.staging.k6.io/docs/refs/pull/1175/merge/

Example: [API Crud Operations Example](https://mdr-ci.staging.k6.io/docs/refs/pull/1175/merge/examples/api-crud-operations/)

### CODE REVIEW

**Review by ppcano — COMMENTED** [review]
This is a good change. The new name and intro define the example more clearly and will help folks find this common FAQ. 

When reviewing, I thought it is important to provide an example using k6 core APIs, `k6/http` and `checks` instead of `httpx` and `k6chaijs`, respectively.  Two suggestions; either rewrite the example or provide two examples.


The reason is that a lot of new users find this example. And the current example uses the advanced and not-so-common APIs: `k6chaijs` and `httpx`. New users should start with k6 core APIS, which are more stable and documented everywhere.

**Comment by immavalls** [comment]
Thanks for the review, @ppcano I think I'll take the suggestion to add two examples. I tend to send users in the forum to https://k6.io/docs/examples/advanced-api-flow/ and the example in https://test-api.k6.io/, and apart from confusing them a bit, it seems the new APIs make the example more readable. I think it's great if we showcase both ways, so I'll definitely adapt the older example to a full CRUD and add it as a first example.

**Comment by immavalls** [comment]
@ppcano Can you review it again and let me know if you would change anything else? 🙇‍♀️

File: src/data/markdown/docs/05 Examples/01 Examples/10 api-crud-operations.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+The examples showcase the testing of CRUD operations on a REST API. 
+
+CRUD refers to the basic operations in a database: Create, Read, Update, and Delete. We can map these operations to HTTP methods in REST APIs:
+
+- _Create_: HTTP `POST` operation to create a new resource.
+- _Read_: HTTP `GET` to retrieve a resource.
+- _Update_: HTTP `PUT`or `PATCH` to change an existing resource.
+- _Delete_: HTTP `DELETE` to remove a resource.
+
+You'll find two test examples: the first one uses the core k6 APIs (`k6/http` and `checks`), and the second showcases more recent APIs (`httpx` and `k6chaijs`).
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/05 Examples/01 Examples/10 api-crud-operations.md:15]:
```suggestion
This document has two examples, one that uses the core k6 APIs (`k6/http` and `checks`) and another to show the more recent APIs (`httpx` and `k6chaijs`).
```

**Review by MattDodsonEnglish — APPROVED** [review]
LGTM! Some tiny nits in the intro.

**Review by ppcano — APPROVED** [review]

**Pull request merged**